examples/rmg/superminimal/simulation_flux_rop_superminimal_RMS.ipynb

```diff
@@ -318,8 +383,25 @@
     }
    ],
    "source": [
+    "t = 0.45\n",
     "rms.plotradicalrops(sim, t)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "94611a10",
+   "metadata": {},
+   "source": [
+    "## Next step"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "269dc12b",
+   "metadata": {},
+   "source": [
+    "One of the highlights of RMS is have several sensitivity algorithms available, and some are really cost-efficient running with in seconds even for large systems. It is also possible to run some complex systems, e.g., multiple-phase systems in RMS. Please join us and learn more about RMS in the \""
```

Comment:
Can you rewording the "Please join us..." sentence? Something like "You can find notebooks about more complicated systems and sensitivities in the same directory. You can follow recorded RMG workshop lectures to learn and practice the more advanced features of RMS. (attach some link)" Since we don't have a Youtube playlist for the 2023 RMG workshop. Let's just mention the resources page on the RMG website.